vunit/test_suites.py

```diff
@@ -171,6 +171,11 @@ def run(self, output_path, read_output):
 
         results = self._read_test_results(file_name=get_result_file_name(output_path))
 
+        if hasattr(self._simulator_if, 'get_vhdl_standard') and \

```

Comment:
I would prefer if hasattr was avoided. I also think the simulator_if should provide a method named `has_valid_exit_code` which is default False for all simulators but True for GHDL with vhdl_standard=2008 instead of checking the standard directly here. 

Also why "if name is PASSED" ?

Reply:
> I would prefer if hasattr was avoided. I also think the simulator_if should provide a method named `has_valid_exit_code` which is default False for all simulators but True for GHDL with vhdl_standard=2008 instead of checking the standard directly here.

Agree.

> Also why "if name is PASSED" ?

At first I just set all to FAILED directly. But that produced errors with the tests that expect to get SKIPPED. Now, only FAILED are rewritten and SKIPPED are kept. I believe that it is not a false positive to return SKIPPED, even if the binary failed.